src/daffodilDebugger/daffodil.ts

```diff
@@ -83,13 +96,15 @@ export class DaffodilDebugEvent<
 > implements DebugSessionCustomEvent
 {
   readonly session: DebugSession
+
   constructor(
     readonly event: E,
     readonly body: B
   ) {
     this.session = debug.activeDebugSession!
   }
-  asEditorMessage(): { command: string; data: any } {
+
+  asObject(): { command: string; data: any } {
```

Comment:
Looks like the `{command: string; data: any}` could have a type definition since this signature is being called in multiple places.